### PR TITLE
Add skill for QA API tests generation

### PR DESCRIPTION
**Pull request by shabrov**
api-test-gen — API Test Generator Skill

Generates a complete, runnable API test project from an API specification or plain endpoint description. Takes any of OpenAPI/Swagger (JSON or YAML), Postman Collection, GraphQL schema, or a verbal description of endpoints as input and produces typed models, test files, project scaffolding, and an HTML coverage report.

**Frameworks supported**

- TypeScript + Playwright
- Python + pytest

**Test coverage per endpoint**

- Happy path — valid inputs, expected 2xx response
- Negative — missing required fields, wrong types, missing auth
- Edge — boundary values from schema constraints (minLength, maximum, enum, etc.)
- Workflow — auto-detected CRUD chains (POST → GET → PUT → DELETE → 404)

### CODE REVIEW

File: registry/skills/api-test-gen/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,357 @@
+---
+name: api-test-gen
+description: >
+  Generate automated API test suites with typed models from OpenAPI/Swagger specs,
+  Postman collections, GraphQL schemas, or plain API descriptions. Produces
+  TypeScript/Playwright or Python/pytest test files, typed request/response models,
```
Comment (AlexanderMakarov) [inline registry/skills/api-test-gen/SKILL.md:6]: >TypeScript/Playwright or Python/pytest test files

Do we really need to deliver it as one skill?  Maybe separate it on two skills?

File: registry/skills/api-test-gen/SKILL.md [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+  full project scaffolding, and an HTML report. Trigger for: "generate tests from swagger",
+  "create API tests from OpenAPI", "generate playwright tests from spec", "generate typed
```
Comment (AlexanderMakarov) [inline registry/skills/api-test-gen/SKILL.md:8]: Please don't try to enumerate all keywords possible here. LLM Is capable to do a semantic search.

File: registry/skills/api-test-gen/SKILL.md [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+7. [Run tests](#step-6--run-tests) — execute and capture results
+8. [Generate HTML report](#step-7--generate-html-report) — human-readable results
+
+Skip steps 7–8 if the user only wants the generated files, not execution.
+
+---
+
+## Step 1 — Ingest Spec
+
+**Supported input types:**
```
Comment (AlexanderMakarov) [inline registry/skills/api-test-gen/SKILL.md:35]: Why we need to list supported input types if there are no:
- scripts/code supporting only listed items,
- prompt instructions hard-coded to listed items

?

File: registry/skills/api-test-gen/references/scaffold.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
+# Project Scaffold Reference
+
+## .env.example (always generate this)
```
Comment (AlexanderMakarov) [inline registry/skills/api-test-gen/references/scaffold.md:3]: This is too hardcoded on the specific project.

**Review by AlexanderMakarov — CHANGES_REQUESTED** [review]
